### PR TITLE
Add confirm in upgrade

### PR DESCRIPTION
**Pull request by Win-Man**
### What problem does this PR solve? 
TiUP will upgrade the cluster directly without confirming when execute upgrade command. Upgrade is an important operation that requied confirm by the user.

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
```
tiup cluster upgrade ${cluster-name} ${version}
```
 - No code

Code changes

 - Has exported function/method change

Side effects

- 

Related changes

 - Need to cherry-pick to the release branch

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cluster/manager.go [hunk 1]
```diff
@@ -815,6 +815,18 @@ func (m *Manager) Upgrade(clusterName string, clusterVersion string, opt operato
...
+	if !skipConfirm {
+		if err := cliutil.PromptForConfirmOrAbortError(
+			"This operation will upgrade %s %s cluster %s to %s.\nDo you want to continue? [y/N]:",
+			m.sysName,
+			color.HiYellowString(base.Version),
+			color.HiYellowString(clusterName),
+			color.HiYellowString(clusterVersion)); err != nil {
+			return err
+		}
+		log.Infof("Destroying cluster...")
```
Comment (9547) [inline pkg/cluster/manager.go:827]: s/Destroying/Upgrading/

Comment (Win-Man) [reply pkg/cluster/manager.go]: Fix it.

**Pull request merged**